website/templates/profile/notifications.mako

```diff
@@ -48,13 +47,16 @@
             </div>
     </div>
 </div>
+</%def>
 
-<script type="text/javascript">
-
-    $script(['/static/js/notificationsConfig.js']);
-    $script.ready('NotificationsConfig', function() {
-        var notifications = new NotificationsConfig('#selectLists', '${website.settings.MAILCHIMP_GENERAL_LIST}');
-    });
-</script>
-
+<%def name="javascript()">
+    <% import website%>
```

Comment:
I tried moving `<% import website%>` to the top of the file, but I get an error that it's undefined.